docs/F-Refinements/pointers-arrays-and-structs.md

```diff
@@ -30,7 +34,7 @@ This chapter examines this relationship between pointers, arrays and structures
 
 We can model RAM as a linear map and use addresses on this map to identify bytes of information stored within memory.  For instance, in 512Mb of RAM, the address 0 identifies the first byte, the address 1 identifies the second byte and address 512Mb-1 identifies the last byte. 
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/memory%20map.png)
+![Representation of RAM](https://ict.senecacollege.ca//~ipc144/pages/images/memory%20map.png)
```

Comment:
This image needs to be brought into the repo vs. linked externally.

Reply:
fixed